pkg/securitypolicy/securitypolicy_test.go

```diff
@@ -1146,47 +1142,45 @@ func selectContainerFromConstraints(constraints *generatedConstraints, r *rand.R
 }
 
 type dataGenerator struct {
-	rng               *rand.Rand
-	mountTargets      map[string]struct{}
-	containerIDs      map[string]struct{}
-	sandboxIDs        map[string]struct{}
-	enforcementPoints map[string]struct{}
+	rng                *rand.Rand
+	mountTargets       map[string]struct{}
```

Comment:
it probably makes sense to define
```go
type stringSet map[string]struct{}

func (s stringSet) In(s string) bool { /* ... */ }
func (s stringSet) Add(s string) bool { /* ... */ }
```

Reply:
I like this, but I don't think it should be done as part of this PR. I'm going to add it to our list of "nits", i.e. things which we will address as part of refactoring once all functionality has been added.